ee/localserver/jwk_keys.go

```diff
@@ -0,0 +1,89 @@
+package localserver
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"math/big"
+)
+
+type jwk struct {
```

Comment:
Do you think this could potentially be worth moving to its own `ee/jwk` package, in case we want to use it elsewhere in the future? (I'm fine with leaving it here, or moving it in a later PR.)

Reply:
I considered that, but my gut feeling is that if were to do an `ee/jwk` we should probably fully implement the jwk spec and at that point my as well use the pkg?